doc/yay.8

```diff
@@ -326,6 +326,14 @@ only GitHub packages are supported.
 .B \-\-nodevel
 Do not check for development packages updates during sysupgrade.
 
+.TP
+.B \-\-needed
+Pass --needed option to pacman by default.
```

Comment:
What does "by default" mean here? Doesn't that only apply if you set it in the config file?

Reply:
Exactly. This is under "PERMANENT CONFIGURATION SETTINGS" so were indeed speaking about the config file

Reply:
They only become permanent when `--save` is used. Without that it will not be by default.